src/components/TeacherComponents/ResourcePageLayout/ResourcePageLayout.tsx

```diff
@@ -115,145 +96,62 @@ const ResourcePageLayout: FC<ResourcePageLayoutProps> = (props) => {
             $gap={16}
             $maxWidth={420}
           >
-            <OakHeading
-              tag="h2"
-              $font={["heading-6", "heading-5"]}
-              $mb={["space-between-m", "space-between-m2"]}
-            >
-              Your details
-            </OakHeading>
-            {props.errors.school && (
-              <FieldError id="school-error">
-                {props.errors.school?.message}
-              </FieldError>
-            )}
-            {props.showNoResources ? (
-              props.page === "download" ? (
+            {props.showNoResources &&
+              (props.page === "download" ? (
                 <NoResourcesToDownload />
               ) : (
                 <NoResourcesToShare />
-              )
-            ) : (
+              ))}
+            {!props.showNoResources && (
```

Comment:
This has been moved up the component tree slightly so that it's logic didn't need to be included in the TermsAgreementFormComponent, see screenshots in the PR.